resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/kubernetes/submit/ClientV2Suite.scala

```diff
@@ -301,11 +301,14 @@ class ClientV2Suite extends SparkFunSuite with BeforeAndAfter {
       APP_NAME,
       APP_RESOURCE_PREFIX,
       APP_ID,
+      "",
```

Comment:
Can you update ClientV2Suite to ensure that we're getting an instance of the file mounter and using it to mount the PySpark files?